packages/twig-language-server/package.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: twig-language-server
+description: Twig Language Server.
+homepage: https://github.com/moetelo/twiggy
+licenses:
+  - MPL-2.0
+languages:
+  - Twig
+  - HTML
+categories:
+  - LSP
+  - Linter
+
+source:
+  # renovate:datasource=github-tags
+  id: pkg:generic/moetelo/twiggy@v0.6.2
+  download:
+    files:
+      twiggy.vsix: https://moetelo.gallery.vsassets.io/_apis/public/gallery/publisher/moetelo/extension/twiggy/{{ version | strip_prefix "v" }}/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage
```

Comment:
This does not seem like a stable URL nor is it allowed to use directly per VSCode marketplace terms of service.

The best option would be if they were to publish their https://github.com/moetelo/twiggy/tree/master/packages/language-server package to npm. It seems like there already is a `twig-language-server` on npm, provided by the upstream fork.

Reply:
Alright, I've asked over there: https://github.com/moetelo/twiggy/issues/5

Should I keep this PR open and update it once there is another way to install it? I thought about building it from source but was not sure how to go about it with the way the repository is structured. Would that also be an option?

Reply:
> Should I keep this PR open and update it once there is another way to install it? I thought about building it from source but was not sure how to go about it with the way the repository is structured. Would that also be an option?

Building from source is a last resort when all other options have been exhausted. I'd like to shift responsibility to package maintainers themselves whenever it makes sense as it'd benefit everyone. Let's keep this PR open for the time being, it should be in their interest to publish to npm.

Reply:
@williamboman The package is now published as twiggy-language-server. The maintainer asked to rename the package in this PR. Should I just update here or create a new PR?

Reply:
Cool! Let's update here :)

Reply:
any updates on this?

Reply:
@SushyDev I'm on it, just had no time the last weeks. And now I just noticed that the package contains neither source code nor a built version. I've updated my issue at the source. Sorry for the wait!